lib/kitchen/directions/chapter_review_container/v1.rb

```diff
@@ -4,10 +4,12 @@ module Kitchen::Directions::ChapterReviewContainer
   class V1
     renderable
 
-    def bake(chapter:, metadata_source:)
+    def bake(chapter:, metadata_source:, klass: 'chapter-review')
       @metadata = metadata_source.children_to_keep.copy
+      @klass = klass
+      @title = klass == 'chapter-review' ? I18n.t(:eoc_chapter_review) : I18n.t(:"eoc.#{klass}")
```

Comment:
The ternary statement wouldn't be necessary if `Chapter Review` was stored under `eoc.chapter-review` & could be retrieved the same way as other titles, but I believe this would make this a breaking change. Thoughts?

Reply:
let's store this under eoc, so we can override the title from book specific translations